### ISSUE TITLE
Bitmart encoding issue

### ISSUE THREAD

**Issue by molockh** [ccxt/ccxt#10538]
Previously, there was no such error, but today there is a problem with the Bitmart exchange. 
I call the async fetch_ticker function and I get the UnicodeDecodeError. 
The only problem is with the async library. The synchronous version works fine.

- OS: MacOS Big Sur
- Programming Language version: Python 3.8
- CCXT version: 1.60.84

```
async def main():
    exchange = ccxt.bitmart()
    ticker = await exchange.fetch_ticker("BTC/USDT")
    print(ticker)

if __name__ == "__main__":
    loop = asyncio.get_event_loop()
    loop.run_until_complete(main())
```

```
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xa0 in position 15046: invalid start byte
```

**Comment by frosty00** [comment]
I can't reproduce

**Comment by fooberichu150** [comment]
> I can't reproduce

Still very much reproducible for me using ccxt Python library in conjunction with freqtrade. If I use the Javascript library instead, it works just fine. I've tested with CCXT version: 1.59.2, 1.60.11, and 1.60.68.

**Comment by frosty00** [comment]
```
    response = await self.publicAccountGetCurrencies(params)
  File "/Users/carlorevelli/code/ccxt/python/ccxt/async_support/base/exchange.py", line 97, in fetch2
    return await self.fetch(request['url'], request['method'], request['headers'], request['body'])
  File "/Users/carlorevelli/code/ccxt/python/ccxt/async_support/base/exchange.py", line 123, in fetch
    http_response = await response.text()
  File "/Users/carlorevelli/code/ccxt/venv/lib/python3.8/site-packages/aiohttp/client_reqrep.py", line 1081, in text
    return self._body.decode(encoding, errors=errors)  # type: ignore
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xa0 in position 15046: invalid start byte
```

ok i can reproduce

**Comment by frosty00** [comment]
it's in the async, i guess the encoding is wrong

**Comment by kroitor** [comment]
@frosty00 i think it's wrong in the content-encoding HTTP header from Bitmart, cause it worked a few days ago, and then stopped working, and we have not changed anything that would break it in our code. I'm not sure if it's worth adding a workaround for all exchanges, just to fix this specific one. Would be great if we can find a way of fixing Bitmart specifically without affecting the other code.

**Comment by kroitor** [comment]
On a second thought, the `aiohttp` implementation should match the `requests` implementation...

**Comment by frosty00** [comment]
it's a problem inside the decoding of the aiohttp response

**Comment by kroitor** [comment]
['ATTRS', '__aenter__', '__aexit__', '__class__', '__del__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__', '_body', '_cache', '_cleanup_writer', '_closed', '_connection', '_content_dict', '_content_type', '_continue', '_headers', '_history', '_loop', '_notify_content', '_parse_content_type', '_protocol', '_raw_headers', '_real_url', '_released', '_request_info', '_response_eof', '_session', '_source_traceback', '_stored_content_type', '_timer', '_traces', '_url', '_writer', 'charset', 'close', 'closed', 'connection', 'content', 'content_disposition', 'content_length', 'content_type', 'cookies', 'get_encoding', 'headers', 'history', 'host', 'json', 'links', 'method', 'ok', 'raise_for_status', 'raw_headers', 'read', 'real_url', 'reason', 'release', 'request_info', 'start', 'status', 'text', 'url', 'url_obj', 'version', 'wait_for_close']

I've ran print(dir(response)) ↑ they have these standalone things like `get_encoding` and `content_type`.

**Comment by kroitor** [comment]
```
Request: GET https://api-cloud.bitmart.com/account/v1/currencies None None
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xa0 in position 15047: invalid start byte
```

This is the problem:

- https://api-cloud.bitmart.com/account/v1/currencies

<img width="1063" alt="Screenshot 2021-11-10 at 05 31 26" src="https://user-images.githubusercontent.com/1294454/141038921-9d181373-355a-4f04-bc13-3ed2b28838cf.png">

**Comment by kroitor** [comment]
Will try to add a workaround for that.

**Comment by kroitor** [comment]
Fixed in CCXT 1.60.86+, the new build will arrive in ~20 minutes.
